app/models/commit.rb

```diff
@@ -28,6 +28,13 @@ class Commit
   validates :message, uniqueness: {:scope => :commit_date}
 
   index({ user_id: 1 })
+  scope :in_range, -> (from, to) {
+    where(:commit_date.gte => from, :commit_date.lte => to) if from.presence && to.presence
```

Comment:
Remove this if condition. The query should raise exception if no dates are passed